scootapi/setup/workers.go

```diff
@@ -0,0 +1,95 @@
+package setup
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+)
+
+// WorkersStrategy is a strategy to start workers and returns the config to pass to a scheduler to talk to them
+type WorkersStrategy interface {
+	// StartupWorkers should startup necessary workers, and returns the config to pass to Scheduler, or an error
+	StartupWorkers() (string, error)
+}
+
+// InMemoryWorkersStrategy will use in-memory workers (to test the Scheduler logic)
+type InMemoryWorkersStrategy struct {
+	workersFlag string
+}
+
+// NewInMemoryWorkers creates a new InMemoryWorkersStartup
+func NewInMemoryWorkers(workersFlag string) *InMemoryWorkersStrategy {
+	return &InMemoryWorkersStrategy{workersFlag: workersFlag}
+}
+
+func (s *InMemoryWorkersStrategy) StartupWorkers() (string, error) {
+	log.Println("Using in-memory workers")
+	if s.workersFlag == "" {
+		return "local.memory", nil
+	}
+
+	return fmt.Sprintf(`{"Cluster": {"Type": "memory", "Count": %s}, "Workers": {"Type": "local"}}`, s.workersFlag), nil
```

Comment:
It's not a strategyName, it's what gets passed to the scheduler per se, which is built-in to the scheduler (because of Caitie's config changes in the scheduler)